src/store/index.js

```diff
@@ -1,25 +1,41 @@
-import ReducerRegistry from '@redhat-cloud-services/frontend-components-utilities/ReducerRegistry';
+// import ReducerRegistry from '@redhat-cloud-services/frontend-components-utilities/ReducerRegistry';
+import { createContext } from 'react';
+import { ReducerRegistry } from '@redhat-cloud-services/frontend-components-utilities/ReducerRegistry';
+import { applyReducerHash } from '@redhat-cloud-services/frontend-components-utilities/ReducerRegistry';
```

Comment:
You can join these two imports togethere
```suggestion
import { applyReducerHash, ReducerRegistry } from '@redhat-cloud-services/frontend-components-utilities/ReducerRegistry';
```